src/test/java/rx/subscriptions/CompositeSubscriptionTest.java

```diff
@@ -344,4 +344,121 @@ public void testAddingNullSubscriptionIllegal() {
         csub.add(null);
     }
 
+    @Test
+    public void testAddAll() {
+        final AtomicInteger counter = new AtomicInteger();
+        CompositeSubscription s = new CompositeSubscription();
+        s.addAll(new Subscription() {
+            @Override
+            public void unsubscribe() {
+                counter.incrementAndGet();
+            }
+
+            @Override
+            public boolean isUnsubscribed() {
+                return false;
```

Comment:
not sure if it's correct behavior to always return `false` here. Maybe create `TestSubscription` with `AtomicBoolean unsubscribed`?